src/server.test.ts

```diff
@@ -429,3 +430,34 @@ test('Returns a signed URL or the blob directly based on the request parameters'
   await server.stop()
   await fs.rm(directory.path, { force: true, recursive: true })
 })
+
+test('Accepts stores with `experimentalRegion: "context"`', async () => {
```

Comment:
this test name sounds a little too deep in test specifics - is `context` a special region name? Judging from the test, here's a name that would've surprised me less:
```suggestion
test('Accepts stores with `experimentalRegion`', async () => {
```